client/src/pages/account/teams/index.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { DrillDownNav } from 'components/DrillDownNav'
+import { AccountEmptyPage } from 'components/AccountEmptyPage'
+import EmptyTeams from '../../../images/empty-teams.svg'
+
+const Teams = () => {
```

Comment:
Same question here with teams.

Reply:
this is consistent with the name of the url